src/legacy/upload/test/e2e/index.html

```diff
@@ -0,0 +1,186 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>vl-upload voorbeeld</title>
+    <script type="module" src="../../../../../build/index.js"></script>
+
+    <link rel="stylesheet" type="text/css" href="../../../../../lib/components/body/styles.css" />
+    <link rel="stylesheet" type="text/css" href="../../../../../lib/components/button/styles.css" />
+  </head>
+
+  <body is="vl-body">
+    <vl-demo-page
```

Comment:
Alle `vl-demo-page` en `vl-demo` tags mogen eruit, omdat die niet bestaan bij ons. Als ik de test pagina open zie ik ook geen enkele upload component, hoe kunnen de testen slagen?

Reply:
![Schermafbeelding 2022-03-18 om 14 43 32](https://user-images.githubusercontent.com/22538537/159013916-238b9af1-519e-41a9-8018-254db8c0b22a.png)

Reply:
Ik merk ook dat er een file van 3mb wordt toegevoegd in de testen. Dit kunnen we echt niet maken, de test folder wordt mee gepackaged (voorlopig) en is dus een zeer slecht idee. Ofwel verwijs je in je testen naar een url die een file download van 3mb, ofwel disable je tijdelijk de test.